doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1103,3 +1117,4 @@ Documentation
     The instructions for using Programmer were moved to the :ref:`ug_nrf9160` page.
   * Replaced LTE Link Monitor and Trace Collector apps with `nRF Connect Serial Terminal`_ and `Cellular Monitor`_ apps.
   * Renamed nRF91 AT Commands Reference Guide to `nRF9160 AT Commands Reference Guide`_, and added references to the `nRF91x1 AT Commands Reference Guide`_ in the documentation.
+  * All references to GNSS assistance from ``A-GPS`` to `A-GNSS`_.
```

Comment:
This is not really about documentation change but mainly about the update in the code, which of course reflects to the documentation. So we don't have a better place in the release notes to tell people about sdk-nrf wide replacement of a terminology?
@peknis

Reply:
The closest thing I could find is "Working with nRF91 Series". Not sure what to say there, though.